index.html

```diff
@@ -70,30 +70,30 @@
 	<div style="display:none" id="footer">
 		<div id="sound" sound="on" class="no-select">
 			<div id="sound_icon"></div>
-			<div id="sound_on">ON</div> <!-- TRANSLATE THIS -->
-			<div id="sound_off">OFF</div> <!-- TRANSLATE THIS -->
+			<div id="sound_on"></div> <!-- TRANSLATE THIS -->
+			<div id="sound_off"></div> <!-- TRANSLATE THIS -->
 		</div>
 		<div id="select"></div>
 
 		<!-- TRANSLATE "title" & "text". Make sure it's less than 112 characters, for Twitter's sake -->
-		<!-- Also, change "link" to whatever the link for the translated version will be! -->
-		<sharing title="The Evolution of Trust"
-				 text="People no longer trust each other. Why? And how can we fix it? An interactive guide to the game theory of trust:"
-				 link="http://ncase.me/trust/"></sharing>
+		<!-- Also, "link" will be changed later to whatever the link for the translated version will be. -->
+		<sharing title="Ewolucja Zaufania"
+				 text="Ludzie już sobie nie ufają. Dlaczego? Jak możemy to naprawić? Interaktywny przewodnik po zaufaniu według teorii gier:"
+				 link="http://sin.github.io/trust/"></sharing>
 
 		<!-- List of translations thus far! -->
 		<div id="translations">
 			fan-translations so far:
 			<a href="https://github.com/ncase/trust#how-to-translate-this-thang" target="_blank">(add your own!)</a> |
 			<a href="https://ccamara.github.io/trust/">Español de España</a> |
-			<a href="https://ayowel.github.io/trust/">Français</a> | 
+			<a href="https://ayowel.github.io/trust/">Français</a> |
 			<a href="https://jkoelling.github.io/trust/">Deutsch</a> |
 			<a href="https://lvdt.github.io/trust/">Italiano</a> |
 			<a href="https://notdotteam.github.io/trust/">Русский</a> |
-			<a href="https://brunolemos.github.io/trust/">Português do Brasil</a> | 
+			<a href="https://brunolemos.github.io/trust/">Português do Brasil</a> |
 			<a href="https://osaatcioglu.github.io/trust/">Türkçe</a> |
-			<a href="http://sekai.co/trust/">中文</a> | 
-			<a href="https://audreyt.github.io/trust-zh-TW/">中文（臺灣）</a>	
+			<a href="http://sekai.co/trust/">中文</a> |
+			<a href="https://audreyt.github.io/trust-zh-TW/">中文（臺灣）</a>
 		</div>
 
 	</div>
```

Comment:
Update to inna sprawa, tutaj na razie zajmijmy się korektą. :)

Ale ogólnie 👍  ;)